meter.go

```diff
@@ -0,0 +1,128 @@
+package metrics
+
+import (
+	"net/http"
+	"sync"
+	"time"
+
+	"gopkg.in/vinxi/layer.v0"
+)
+
+// PublishInterval defines the amount of time to wait between metrics publish cycles.
+// Defaults to 15 seconds.
+var PublishInterval = 15 * time.Second
```

Comment:
It is, actually. You can do: `metrics.PublishInterval = 5 * time.Second`.

I would like to support a `Config` struct to pass instance specific configuration, maybe in the next set of improvements.